commerce_coordinator/apps/commercetools/clients.py

```diff
@@ -269,3 +271,36 @@ def get_product_variant_by_course_run(self, cr_id: str) -> Optional[CTProductVar
             return None
 
         return matching_variant_list[0]
+
+    def update_return_payment_state_after_successful_refund(self, order_id: str,
+                                                            order_version: str, return_item_id: str) -> CTOrder:
+        """
+        Update paymentState on the LineItemReturnItem attached to the order.
+        Updated by the Order ID (UUID)
+
+        Args:
+            order_id (str): Order ID (UUID)
+            order_version (str): Current version of order
+            return_item_id (str): LineItemReturnItem ID
+
+        Returns (CTOrder): Updated order object or
+        Returns Exception: Error if update was unsuccesful.
+        """
+        try:
+
+            return_payment_state_action = OrderSetReturnPaymentStateAction(
+                return_item_id=return_item_id,
+                payment_state=ReturnPaymentState.REFUNDED
+            )
+
+            updated_order = self.base_client.orders.update_by_id(
```

Comment:
Have you verified what happens if send this update call again for an order which is already refunded?

Reply:
No I have not been able to as this is the last step in the automatic refunds process and as such made testing this tricky. 
I could map through the order's return item array to find the return item we wish to update and do a check for it's current value to prevent us from updating if that value is already set to refunded?

Reply:
I agree. The CommercetoolsError exception should catch this case